pkg/sidecar/docker_reactor.go

```diff
@@ -31,22 +31,24 @@ import (
 var PublicAddr = net.ParseIP("1.1.1.1")
 
 type DockerReactor struct {
-	client  *sync.Client
-	routes  []net.IP
-	manager *docker.Manager
+	client         *sync.Client
+	servicesRoutes []net.IP
+	manager        *docker.Manager
 }
 
 func NewDockerReactor() (Reactor, error) {
 	// TODO: Generalize this to a list of services.
 	wantedRoutes := []string{
 		os.Getenv(EnvRedisHost),
+		os.Getenv(EnvInfluxdbHost),
 	}
 
 	var resolvedRoutes []net.IP
 	for _, route := range wantedRoutes {
 		ip, err := net.ResolveIPAddr("ip4", route)
 		if err != nil {
-			return nil, fmt.Errorf("failed to resolve host %s: %w", route, err)
+			logging.S().Warnw("failed to resolve host", "host", route, "err", err.Error())
```

Comment:
Change in behaviour here - we no longer return an error if we can't resolve the service, allowing for `optional` services.